android/app/src/main/res/values-eu/strings.xml

```diff
@@ -603,7 +603,7 @@
     <string name="routing_add_start_point">Erabili bilaketa edo sakatu mapan ibilbidearen abiapuntua gehitzeko</string>
     <!-- User selected the start of a route by pressing Route From. Now the destination of a route should be selected using search or by tapping on the map and then pressing "Route To". -->
     <string name="routing_add_finish_point">Erabili bilaketa edo sakatu mapan helmuga puntu bat gehitzeko</string>
-    <string name="placepage_remove_stop">Kendu</string>
+    <string name="placepage_remove_stop">Kendu geldiunea</string>
```

Comment:
Kendu geralekua

Reply:
@alexgabi: `Kendu geltokia` seems to translate as "Remove station", while `Kendu geralekua` (as proposed by @andergd ) translates as "Remove stop", which fits better to the English version. Can you please confirm?

Reply:
In Basque language "geltokia" is used to say "station" and "stop":

**geltoki**

    1  iz. [station](https://hiztegiak.elhuyar.eus/en_eu/station)

    tren-geltokia: railway station

    2  iz. [stop](https://hiztegiak.elhuyar.eus/en_eu/stop)

    autobus-geltokia: bus stop

Reply:
Hi @alexgabi. Finally, this PR has been merged with the string `Kendu geralekua`.

If you __really__ think that this is not correct, please let us know so we can launch an additional PR to fix it.

Please note that right now we have 2 strings containing the word "Stop" in the same context, and that we must use the same noun on both of them:

```
[placepage_remove_stop]
  en = Remove Stop
  eu = Kendu geralekua

[placepage_add_stop]
  en = Add Stop
  eu = Gehitu geldiunea
```

Please, let us know clearly which 2 strings we shall finally use.

Reply:
The feeling the word "geltoki" gives to me is as a stop as in a bus route. The word "geraleku" feels like a place that can be used to stop or rest. Any of both could be used with the nuance of each of them, in my opinion. Maybe in the context of a route, the word "geltoki" is more suitable, but "geraleku" seems fine to me too. However, the word "geldiune" refers to a _time_ to rest instead of a _place_, such as a break or a pause, so I would suggest not using it.

Reply:
Thanks @andergd, @alexgabi. I'm glad that you 2 both agree, but I honestly still don't know if we should change the strings.

And if we need to change the strings, please state clear crystal which should be the final strings by replacing the blanks with the strings, so I can easily copy and paste them:

```
[placepage_remove_stop]
  en = Remove Stop
  eu = __________________

[placepage_add_stop]
  en = Add Stop
  eu = __________________
```

Reply:
For sake of simplicity, I would suggest using the shorter of both, which is "geltoki", for better visualisation in the app:

```
[placepage_remove_stop]
  en = Remove Stop
  eu = Kendu Geltokia

[placepage_add_stop]
  en = Add Stop
  eu = Gehitu Geltokia
```

Sorry for the inconveniences

Reply:
Thanks @andergd: I've launched [PR 9596](https://github.com/organicmaps/organicmaps/pull/9596) with your proposal. Please check there that everything is correct...